README.md

```diff
@@ -1,30 +1,60 @@
+4. Automatically prune branches that do not have remote branches.
+5. Have the ability to manually add repositories for synchronization from FossConnect project.
+6. Have some short meaningfull output in the UI and detailed logs in a file. There should be a dedicated buttons: 'View logs' to open log folder for convinience, 'View config' to open folder with it.
+7. Build selected projects.
+8. Run selected projects.
+9. Introduce profiles that will contain list of repositories that should be checked for synchronization.
+10. Have the ability to start the tool from the command line passing the profile name as a parameter.
 
 ### Implementation
 
-This will be a desktop application developed using WPF. The configuration file mentioned earlier will be local in the first iteration, in the future it is planned to load it from a remote repository. 
-It is planned to use the git application to work with repositories because the pool of scheduled tasks does not require complex commands. List of commands that are planned to be used are : branch, checkout, pull, stash. 
-All information will be presented in a table. One line in it corresponds to one project. 
-To select the necessary branch there will be a ComboBox with an available selection. After branch selection, SourceExplorer will check if the branch is presented locally. If not - it will perform a pull operation. 
-Also, there will be a ComboBox for build mode.
-For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button.
+This will be a desktop application developed using WPF targeting .NET 6. It is planned to use the libgit2shart nuget package to work with repositories because this library is time-tested and will allow you to perform all planned operations and expand their list in the future. List of commands that are planned to be used now are : branch, checkout, pull, stash.
+
+All information will be presented in the table. One line in it corresponds to one project.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/mainwindow.png)
+
+To select the necessary branch there will be a ComboBox with an available selection. 
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/branchselecting.png)
+Also, there will be a ComboBox for build mode. <br />
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/buildmodeselecting.png)  <br />
+For each operation, buttons are provided to perform it. If action is required to be performed on multiple projects, user must click on corresponding CheckBoxes and then on an action button in header of the table.
+![Image alt](https://github.com/BelehovEgor/FossHelperUI/raw/design/Images/projectchoice.png)
+Special buttons are located on the menu bar at the top of the window.
```

Comment:
"Select root folder"
and the name of the menu should be "File"
![image](https://user-images.githubusercontent.com/42497175/143389717-d621ceba-623e-439b-bc8f-0a2d92dfea91.png)

Reply:
Logs and Config menu items should be also in the "File" menu.

Reply:
Also, we will need the menu item for the 'Init' function. And it should have a Tooltip "Download from remote repositories" because many people will forget what 'Init' actually does )

Reply:
fixed, but I have problems with tip. ShowDuraction is too small and I can't change it